src/controller/postRouter.js

```diff
@@ -31,18 +31,37 @@ postRouter.get("/", async (req, res) => {
     }
 });
 
-postRouter.patch("/replies", authenticate, validateTextBody, async (req, res) => {
+postRouter.patch("/:id/replies", authenticate, postMiddleware.validateTextBody, async (req, res) => {
     //TODO check song title exists in API
     try {
-        await createReply(res.locals.user.username, req.body.text, req.body.id);
+        await postService.createReply(res.locals.user.itemID, req.body.text, req.params.id);
         res.status(200).json({
-            message: "Reply successfully created"
+            message: `Replied to ${req.params.id} successfully`
```

Comment:
I would send back the created reply object so we can grab the replyID in postman for easier testing. mostly if we want to allow replies to be edited or deleted

Reply:
Good idea